libs/MotionKit/source/MotionKit.cpp

```diff
@@ -106,7 +111,7 @@ void MotionKit::run()
 
 auto MotionKit::mapSpeed(float speed) const -> float
 {
-	return utils::math::map(speed, 0.F, 1.F, kMinimalViableRobotPwm, kPwmMaxValue);
+	return utils::math::map(speed, 0.F, 1.8F, kMinimalViableRobotPwm, kPwmMaxValue);
```

Comment:
why this change? you'll get PWM values > 1.0, no?

Reply:
Oh, you should set it to the maximum value of PID if it is instead of magic number `1.8F`